content/ai/concepts/neural-networks/terms/binary-step-activation-function/binary-step-activation-function.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Binary Step Activation Function'
+Description: 'A binary-step function is the simplest threshold based activation function which works by either firing the node in case the threshold value is surpassed or doing nothing at all.'
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Machine Learning'
+  - 'Neural Networks'
+  - 'Functions'
+CatalogContent:
+  - 'machine-learning'
+  - 'path/data-science'
+---
+
+## Binary Step Activation Function
+
+**Binary Step** is the simplest threshold based activation function which works by either activating the node i.e., passing on its output to the next layer in case the threshold value is surpassed or doing nothing at all. 
```

Comment:
```suggestion
**Binary Step** is the simplest threshold-based activation function which works by either activating the node, i.e. passing on its output to the next layer in case the threshold value is surpassed, or doing nothing at all. 
```